core/src/main/java/com/radixdlt/p2p/transport/PeerServerBootstrap.java

```diff
@@ -149,9 +149,13 @@ public void start() throws InterruptedException {
     serverBind = serverBootstrap.bind(config.listenAddress(), config.listenPort()).sync();
   }
 
-  public void stop() throws InterruptedException {
+  public void stop() {
     if (serverBind != null) {
-      serverBind.channel().close().sync();
+      try {
+        serverBind.channel().close().sync();
+      } catch (InterruptedException e) {
+        Thread.currentThread().interrupt();
```

Comment:
(just noting: here I understand that it actually makes sense to silently ignore the interrupt [i.e. acknowledge that a graceful `.stop()` was interrupted by the user furiously doing more `ctrl+C`], and set the flag back up [i.e. because other consecutive spin-down methods may also like to skip the gracefulness in such case], without propagating exception [because that `.stop()` is not critical in interrupt scenario] - exactly what you did 👍 )